lib/xboxrt/libc_extensions/string_ext_.h

```diff
@@ -9,6 +9,27 @@ static char *_strdup (const char *s)
     return strdup(s);
 }
 
+// Compared to their Win32 counterparts, these functions currently don't invoke
+// an invalid parameter handler routine as described in the MSDN, and instead
+// behave is if the invalid parameter handler allowed to continue execution.
+int _strnicmp (const char *s1, const char *s2, size_t n);
+int _stricmp (const char *s1, const char *s2);
+
+static int strnicmp (const char *s1, const char *s2, size_t n)
+{
+    return _strnicmp(s1, s2, n);
+}
+
+static int stricmp (const char *s1, const char *s2)
+{
+    return _stricmp(s1, s2);
+}
+
+#ifndef NLSCMP_DEFINED
+#define _NLSCMPERROR 2147483647
```

Comment:
nit: `0x7FFFFFFF` for readability?

Reply:
Oh yeah, that's probably better. Changed.